.github/workflows/semgrep-rules-test-develop.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   # Note: if you change this test there will likely need to be a
   # corresponding change in returntocorp/semgrep
-  test-latest:
+  test-develop:
```

Comment:
Why did this get changed? Not blocking, just wondering

Reply:
Oh, so that it had a different name in the GitHub UI. I forgot that I made this change here.